### PR TITLE
Make Che logging variables configurable via helm

### PR DESCRIPTION
**Pull request by Ido-Itz**
### What does this PR do?

This PR allows for che6 helm installations to specify a value for the CHE_LOGGER_CONFIG env variable which allows setting different log levels of different packages.
It also allows to specify a name of a custom appender using CHE_LOGS_APPENDERS_IMPL. as is documented in the following page: https://www.eclipse.org/che/docs/logging.html

### CODE REVIEW

**Comment by riuvshin** [comment]
Can one of the admins verify this patch?

**Comment by riuvshin** [comment]
Can one of the admins verify this patch?

**Comment by riuvshin** [comment]
Can one of the admins verify this patch?

**Comment by skabashnyuk** [comment]
ci-test

**Comment by riuvshin** [comment]
ci-test build report:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/648/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/648//Selenium_20tests_20report/)
[selenium tests report data](http://tests.codenvycorp.com/selenium_test_report_data/10912)
docker image: eclipseche/che-server:10912
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

File: deploy/kubernetes/helm/che/templates/configmap.yaml [hunk 1]
```diff
@@ -70,7 +70,14 @@ data:
   CHE_INFRA_KUBERNETES_INGRESS_ANNOTATIONS__JSON: '{"kubernetes.io/ingress.class": "nginx", "{{ .Values.global.ingressAnnotationsPrefix }}ingress.kubernetes.io/rewrite-target": "/","{{ .Values.global.ingressAnnotationsPrefix }}ingress.kubernetes.io/ssl-redirect": "false","{{ .Values.global.ingressAnnotationsPrefix }}ingress.kubernetes.io/proxy-connect-timeout": "3600","{{ .Values.global.ingressAnnotationsPrefix }}ingress.kubernetes.io/proxy-read-timeout": "3600"}'
 {{- end }}
   CHE_INFRA_KUBERNETES_SERVER__STRATEGY: {{ .Values.global.serverStrategy }}
+{{- if .Values.global.log.loggerConfig }}
+  CHE_LOGGER_CONFIG: {{ .Values.global.log.loggerConfig }}
+{{- end }}
+{{- if .Values.global.log.customAppenderName }}
+  CHE_LOGS_APPENDERS_IMPL: {{ .Values.global.log.customAppenderName }}
+{{- else }}
```
Comment (garagatyi) [inline deploy/kubernetes/helm/che/templates/configmap.yaml]: Optional: Consider setting `plaintext` appender as default value and removing `else` clause to simplify file

**Review by garagatyi — APPROVED** [review]
Looks good!

**Review by sleshchenko — APPROVED** [review]

File: deploy/kubernetes/helm/che/templates/configmap.yaml (hunk 1 above)
Comment (Ido-Itz) [reply deploy/kubernetes/helm/che/templates/configmap.yaml]: Good suggestion. Done.

**Comment by riuvshin** [comment]
ci-test build report:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/649/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/649//Selenium_20tests_20report/)
[selenium tests report data](http://tests.codenvycorp.com/selenium_test_report_data/10912)
docker image: eclipseche/che-server:10912
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

File: deploy/kubernetes/helm/che/templates/configmap.yaml [hunk 2]
```diff
@@ -70,7 +70,10 @@ data:
   CHE_INFRA_KUBERNETES_INGRESS_ANNOTATIONS__JSON: '{"kubernetes.io/ingress.class": "nginx", "{{ .Values.global.ingressAnnotationsPrefix }}ingress.kubernetes.io/rewrite-target": "/","{{ .Values.global.ingressAnnotationsPrefix }}ingress.kubernetes.io/ssl-redirect": "false","{{ .Values.global.ingressAnnotationsPrefix }}ingress.kubernetes.io/proxy-connect-timeout": "3600","{{ .Values.global.ingressAnnotationsPrefix }}ingress.kubernetes.io/proxy-read-timeout": "3600"}'
 {{- end }}
   CHE_INFRA_KUBERNETES_SERVER__STRATEGY: {{ .Values.global.serverStrategy }}
-  CHE_LOGS_APPENDERS_IMPL: "plaintext"
+{{- if .Values.global.log.loggerConfig }}
+  CHE_LOGGER_CONFIG: {{ .Values.global.log.loggerConfig }}
```
Comment (guydc) [inline deploy/kubernetes/helm/che/templates/configmap.yaml:74]: Please add a relevant Deployment Container Env reference to this configmap key

Comment (Ido-Itz) [reply deploy/kubernetes/helm/che/templates/configmap.yaml:74]: Done

**Comment by skabashnyuk** [comment]
ci-test

**Comment by riuvshin** [comment]
ci-test build report:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/659/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/659//Selenium_20tests_20report/)
[selenium tests report data](http://tests.codenvycorp.com/selenium_test_report_data/10912)
docker image: eclipseche/che-server:10912
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Pull request merged**